plasmapy/simulation/particle_tracker.py

```diff
@@ -547,6 +632,11 @@ def load_particles(
         self.v = v.to(u.m / u.s).value
 
     def _stop_particles(self, particles_to_stop_mask):
+        """Stop tracking the particles specified by the stop mask.
+
+        This is accomplished by setting the particle's velocity to NaN.
```

Comment:
"This is represented by setting..." 

Just to be clear that this is a convention we are using.